setup.py

```diff
@@ -53,7 +54,7 @@
     extras_require=EXTRAS_REQUIRE,
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 5 - Production/Stable',
```

Comment:
What does this entail?

Reply:
This is something that likely should have been changed a long time ago. This is just metadata on the package. As this package is used by GA libraries we have been supporting it as a production/stable interface.